src/burn/engine/detect.cpp

```diff
@@ -324,7 +336,7 @@ static HRESULT DownloadUpdateFeed(
     __in_z LPCWSTR wzBundleId,
     __in BURN_USER_EXPERIENCE* pUX,
     __in BURN_UPDATE* pUpdate,
-    __out_opt LPWSTR* psczTempFile
+    __deref_inout_z LPWSTR* ppwzTempFile
```

Comment:
It's a pointer to a pointer to a non-constant wide string that is zero terminated. To me ppwz made sense, not pscz.